src/unitxt/metrics.py

```diff
@@ -15,6 +15,7 @@
 import numpy
 import numpy as np
 import pandas as pd
+import spacy
```

Comment:
```suggestion
```
imports should be done in the `prepare` to avoid adding more dependencies for all the users